athena-jdbc/test-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "athena_work_group" : "FederationIntegrationTests",
+  "spill_bucket" : "",
+  "vpc_id" : "",
+  "security_group_id" : "",
+  "subnet_ids" : [],
+  "availability_zones" : []
```

Comment:
should add the username, password and an option SecretsManager secret ARN here instead of in code.

Reply:
Added params to config file (will tackle SecretsManager in a separate PR).